README.md

```diff
@@ -197,6 +197,16 @@ const App = () => (
 );
 ```
 
+#### Restore default browser navigation for one link
+
+The router intercepts `<a />` links that point to app pages. When you click such a link, it switches the route without loading a full page. It's quite common for users to want to opt-out of this behavior. Here are the rules for making `preact-iso` ignore a link:
+
+1. You're on `https://example.org` but the link points to `https://example.com`.
```

Comment:
I'd like to make sure "cross-origin" is mentioned, that's a keyword some may ctrl + f for

```suggestion
1. Cross-origin links, i.e., you're on `https://example.org` but the link points to `https://example.com`.
```